src/backend/web/routes/posts.js

```diff
@@ -0,0 +1,53 @@
+require('../../lib/config');
+const express = require('express');
+const { getPost, getPosts } = require('../../utils/storage');
+
+const posts = express.Router();
+
+const env = process.env.NODE_ENV || 'development';
+const port = process.env.PORT || 3000;
+
+/**
+ * addGuidToPost inserts the redis key into the fetched post from redis.
```

Comment:
Let's fix our storage/redis code to just include this instead.  We own all the code, we can change anything to make it fit.